languages/elixir/exercises/concept/processes/.docs/after.md

```diff
@@ -0,0 +1,133 @@
+In Elixir, all code runs inside processes. Elixir processes:
+
+- Should not be confused with system processes.
+- Are lightweight.
+  - It is normal to have an Elixir app that runs hundreds of processes.
+- Have specific use cases.
+  - Processes can:
+    - [Keep global state][getting-started-processes-state].
+    - [Contain failure][getting-started-processes-links].
+    - Allow for concurrent and asynchronous code.
+  - It is also normal to have an Elixir app that doesn't explicitly create new processes, especially if it's a library.
```

Comment:
Not sure these need to be nested so deep, they could be first-level items and still be very helpful

Reply:
Is this better? [`6ced16d` (#1654)](https://github.com/exercism/v3/pull/1654/commits/6ced16d8830aa3047d3b025374bcc2b067c18b25)

Reply:
That's awesome!